miou/tests/fuzz.ml

```diff
@@ -38,6 +38,7 @@ module Ca = struct
       @ [ Relative_distinguished_name.singleton (CN "Ephemeral CA for fuzzer") ])
 
   let cacert_lifetime = Ptime.Span.v (365, 0L)
+  let cacert_serial_number = "\xde\xad\xbe\xef"
```

Comment:
is this needed?

Reply:
We can revert my last commit if you want :+1:.